src/main/python/keymaps.py

```diff
@@ -11,6 +11,7 @@
     ("Norwegian (QWERTY)", norwegian.keymap),
     ("Russian (ЙЦУКЕН)", russian.keymap),
     ("Slovak (QWERTY)", slovak.keymap),
+    ("eurKey (QWERTY)", eurkey.keymap),
```

Comment:
Please keep the list alphabetically sorted and rename the layout to EurKEY so as to match the official naming capitalization.